configyamls/nodocker/otel-config.yaml

```diff
@@ -100,6 +100,7 @@ processors:
             - ec2
             - system
             - docker
+            - ecs
```

Comment:
We should put `ecs` above `docker` since for EC2, there will always be docker.

Reply:
For detectors, order will not matter, this will just add more attributes in the data. But, we can discuss more on this.